lib/gs-plugin-loader.c

```diff
@@ -2620,7 +2620,7 @@ mount_added_cb (GVolumeMonitor *volume_monitor,
 	GsPluginLoaderPrivate *priv = gs_plugin_loader_get_instance_private (plugin_loader);
 	g_autoptr(GDrive) drive = g_mount_get_drive (mount);
 
-	if (!g_drive_is_removable (drive))
+	if (!drive || !g_drive_is_removable (drive))
```

Comment:
Nitpick: In g-s the convention it to use `var != NULL` instead of `!var` (I dislike it but tried to follow when I remembered, as it increases the chances of upstreaming).

Reply:
Will fixup.